pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -450,11 +482,21 @@ func processEvents(image string, logChan <-chan logEntry,
 	min := max * 0.3
 	flushTimer := flextimer.NewRangeTicker(time.Duration(min),
 		time.Duration(max))
+
+	// Metrics publish timer. Publish log metrics every 5 minutes.
+	interval = time.Duration(300 * time.Second)
```

Comment:
Want to define the 300*time.Second as a const at the top of the file? Makes it easier to find.

Reply:
will do

Reply:
done